src/lib/stitching/gravity/stitching.ts

```diff
@@ -166,6 +172,23 @@ export const gravityStitchingEnvironment = (
             })
           },
         },
+        addressConnection: {
+          fragment: gql`
+          ... on Me {
+            __typename
```

Comment:
Another question for @zephraph:
If we remove this fragment, `me` returns `null` unless there is something else being queried. Is this expected? (screenshots 👇🏽 )

<details><summary>Screenshots</summary>

When there is another field requested under `Me` (anything like `initials` here)

![Screen Shot 2021-01-13 at 7 15 02 PM](https://user-images.githubusercontent.com/687513/104527901-e0a06300-55d3-11eb-99c7-005df6f5f285.png)

When that field (`initials`) is removed: 😱 

![Screen Shot 2021-01-13 at 7 15 19 PM](https://user-images.githubusercontent.com/687513/104527919-f31a9c80-55d3-11eb-80d4-e2a289188af6.png)

When a fragment with just `__typename` is added:

![Screen Shot 2021-01-13 at 7 16 28 PM](https://user-images.githubusercontent.com/687513/104527953-06c60300-55d4-11eb-9564-a14194a2c5af.png)




</details>

Reply:
This seems likely to be an issue w/ the version of stitching we have. It could be that somehow the stitched resolvers are embedded in the `Me` resolver and if that doesn't fire then the child resolvers don't have a chance to fire. That said, I haven't specifically seen this before. Maybe @mzikherman has?

Reply:
My understanding of this was that you needed to request _something_ besides the purely stitched-in field (otherwise GraphQL seems to short-circuit the resolving entirely).

Rather than request some random ad-hoc fields (which might result in an unneeded loader call), can use the reserved `__typename` which definitely won't trigger any.